Demo/UIKit/ComponentsViewController.swift

```diff
@@ -109,6 +103,20 @@ internal final class ComponentsViewController: UIViewController, PresenterExampl
 
         present(viewController: alertController, completion: nil)
     }
+    
+    internal func showLoadingIndicator() {
+        let loadingViewController = LoadingViewController()
+        loadingViewController.modalPresentationStyle = .overCurrentContext
+        loadingViewController.modalTransitionStyle = .crossDissolve
+        
+        present(viewController: loadingViewController, completion: nil)
```

Comment:
I like the idea of a spinner as a whole new view controller but the usage seems to result in nested closures.
let's discuss

Reply:
59be98488aa0c6a3248dbec85c634875fccac066